client/package.json

```diff
@@ -44,5 +44,9 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "@types/react-router-dom": "^5.1.7",
+    "@types/styled-components": "^5.1.10"
```

Comment:
Good call. We're not exporting any packages, so the `@types` packages should be in here. We should update the any others as necessary in the future.